// error => something unexpected happens
// humans make mistakes
// handling error

//? error causing scenario
// const a = 4;
// a = 10;
// console.log(a);

// ?handling error
//? put your error prone code to try block
// ?put error management logic in catch block

// const a = 4;

// try {
//   a = 10;
//   console.log(a);
// } catch (error) {
//   console.log(error.message);
// }
